PerformanceTests/Sources/PerformanceTestScript/Util/Benchmark.swift

```diff
@@ -0,0 +1,69 @@
+//
```

Comment:
I think the structs here should be separated into their own standalone files. Also, instead of having `Util/` directory contain everything, might be good to separate it out into `Model/`, `Runners/`, etc. for better organization.